docker-compose-dev.yml

```diff
@@ -1,5 +1,30 @@
 version: '3'
 services:
+  web:
```

Comment:
This is 'web' image isn't required.  It's the original Django instance that we used as base template for this project.

Reply:
Removed and pushed change